specification/openapi.json

```
@@ -195,24 +195,79 @@
             "type": "integer",
             "description": "Time in ms when difference between in-chat transfer and Tx timestamp considered as acceptable"
           },
-          "minNodeVersion": {
-            "type": "string",
-            "description": "Minimal node API version",
-            "example": "0.8.0"
-          },
           "nodes": {
-            "type": "array",
-            "items": {
-              "$ref": "#/components/schemas/NodeInfo"
-            },
-            "description": "Node links for API"
-          },
-          "services": {
             "type": "object",
-            "additionalProperties": {
-              "$ref": "#/components/schemas/Service"
+            "properties": {
+                "list": {
+                    "type": "array",
+                    "items": {
+                      "$ref": "#/components/schemas/NodeInfo"
+                    },
+                    "description": "Node links for API"
+                },
+                "healthCheck": {
+                  "type": "object",
+                  "required": ["normalUpdateInterval", "crucialUpdateInterval", "onScreenUpdateInterval", "threshold"],
+                  "properties": {
+                    "normalUpdateInterval": {
+                      "type": "integer",
+                      "description": "Regular node status update interval in ms"
+                    },
+                    "crucialUpdateInterval": {
+                      "type": "integer",
+                      "description": "Node status update interval when there are no active nodes, in ms"
+                    },
+                    "onScreenUpdateInterval": {
+                      "type": "integer",
+                      "description": "On the node screen, the status update interval in ms"
+                    },
+                    "threshold": {
+                      "type": "integer",
+                      "description": "Permissible height difference between nodes"
+                    }
+                  },
+                },
+                "minVersion": {
+                  "type": "string",
+                  "description": "Minimal node API version",
+                  "example": "0.8.0"
+                }
             }
           },
+          "services": {
+              "type": "object",
```

```suggestion
              "type": "object",
              "required": ["list"],
```

Make `list` required. Can we make `healthCheck` for services required as well?